hrm-service/service-tests/csam-reports.test.ts

```diff
@@ -0,0 +1,187 @@
+const supertest = require('supertest');
+const Sequelize = require('sequelize');
+const app = require('../src/app');
+const models = require('../src/models');
+const mocks = require('./mocks');
+
+//Notes: I am having a hard time importing this from `./case-validation.ts`
```

Comment:
You can just do `import './case-validation';` if you just need to run the code in the module but don't need to use anything it defines